lua/rose-pine.lua

```diff
@@ -460,6 +460,26 @@ local function set_highlights()
 		NvimTreeSpecialFile = { link = "NvimTreeNormal" },
 		NvimTreeWindowPicker = { link = "StatusLineTerm" },
 
+		-- nvim-neotest/neotest
+		NeotestAdapterName = { fg = palette.iris },
+		NeotestBorder = { fg = palette.highlight_med },
+		NeotestDir = { fg = palette.foam },
+		NeotestExpandMarker = { fg = palette.highlight_med },
+		NeotestFailed = { fg = palette.love },
+		NeotestFile = { fg = palette.text },
+		NeotestFocused = { bg = palette.highlight_med, fg = palette.gold },
```

Comment:
Sorry for the nitpick but could you put bg after fg; this is consistent with the rest of the codebase and makes it a little easier to scan

Reply:
No problem at all. Updated here: d4bed48